storage/innobase/handler/ha_innodb.cc

```diff
@@ -6926,10 +6939,17 @@ wsrep_store_key_val_for_row(
 								cs->mbmaxlen),
 							&error);
 				}
-				memcpy(sorted, src_start, true_len);
-				true_len = wsrep_innobase_mysql_sort(
-					mysql_type, cs->number, sorted, true_len,
-					REC_VERSION_56_MAX_INDEX_COL_LEN);
+
+				/* Note that when true_len == 0 it does not mean
+				that src_start == nullptr */
+				if (true_len) {
```

Comment:
I find it unnecessary obfuscation to have a redundant variable `true_len` here. We can simply refer to `key_len` here.